packages/@lwc/engine-core/src/framework/profiler.ts

```diff
@@ -49,6 +49,27 @@ const operationIdNameMapping = [
     'lwc-rehydrate',
 ] as const;
```

Comment:
```suggestion
] as const satisfies Record<OperationId, string>;
```
Just for bonus points.